nistats/design_matrix.py

```diff
@@ -86,7 +86,7 @@ def _cosine_drift(period_cut, frame_times):
     Parameters
     ----------
     period_cut : float
-        Cut period of the low-pass filter (in sec)
+        Cut period of the high-pass filter (in sec)
```

Comment:
We should make the `in sec ` consistent one way or another across these docstrings.